docs/source/getting-started/tabular-data/preparing-microdata.ipynb

```diff
@@ -69,7 +69,7 @@
        "  white-space: pre-wrap;\n",
        "}\n",
        "</style>\n",
-       "<small>shape: (4, 2)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>HWUSUAL</th><th>len</th></tr><tr><td>cat</td><td>u32</td></tr></thead><tbody><tr><td>&quot;(0, 20]&quot;</td><td>6407</td></tr><tr><td>&quot;(20, 40]&quot;</td><td>54209</td></tr><tr><td>&quot;(40, 60]&quot;</td><td>15472</td></tr><tr><td>&quot;(98, inf]&quot;</td><td>119814</td></tr></tbody></table></div>"
```

Comment:
Preference to take some time early in this quarter to do some cleanup, including converting notesbooks to RST: Will make reviews a bit easier.